Shared/Views/CampaignList.swift

```diff
@@ -286,6 +349,32 @@ struct CampaignList: View {
                                 if !HIDDEN_CAMPAIGN_IDS.contains(campaign.id) {
                                     NavigationLink(destination: CampaignView(initialCampaign: campaign)) {
                                         FundraiserListItem(campaign: campaign, sortOrder: fundraiserSortOrder, compact: compactListMode, showShareSheet: .constant(false))
+                                            .contextMenu {
+                                                if shouldShowHeadToHead {
+                                                    Button(action: {
+                                                        showHeadToHeadChoice = campaign
+                                                    }) {
+                                                        Label("Start Head to Head", systemImage: "trophy")
+                                                    }
+                                                }
+                                                if campaign.isStarred {
+                                                    Button(action: {
+                                                        Task {
+                                                            await starOrUnstar(campaign: campaign)
+                                                        }
+                                                    }) {
+                                                        Label("Remove Star", systemImage: "star")
+                                                    }
+                                                } else {
+                                                    Button(action: {
+                                                        Task {
+                                                            await starOrUnstar(campaign: campaign)
+                                                        }
+                                                    }) {
+                                                        Label("Star", systemImage: "star.fill")
```

Comment:
Is there a reason the conditional isn't just on the label since the action is the same in both cases?